### PR TITLE
Remove downloaded placeholder image

### LINKED ISSUE #9783: AMP media player downloads 2 placeholder images.
**Describe the bug**
AMP media player downloads 2 images listed in the placeholder srcset which is bad for performance.

Screenshot 2022-01-11 at 11 22 29

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.bbc.com/hausa/media-59822286.amp
2. Inspect the media player placeholder
3. Go to the Network Tab
4. Reload the page
5. See that there are 2 p0bdk3xv.jpg of different sizes.

**Expected behaviour**
AMP media player should only download 1 image listed in the placeholder srcset.

**Screenshots**

**Desktop (please complete the following information):**

- OS: iOS 11.6.2
- Browser Firefox
- Version 95.0.2

**Smartphone (please complete the following information):**

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Checklist**
- [x] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**
Might be worth to look at this [issue](https://github.com/bbc/simorgh/issues/9762) for more context about placeholder srcset and how to remove the amp-img

### PR DESCRIPTION
**Pull request by MuhammadAHussain**
Resolves https://github.com/bbc/simorgh/issues/9783

**Overall change:**
Removes the unnecessary network call which downloads an image as a placeholder. It now uses an svg BBC logo similar to canonical.

**Code changes:**

- Remove url as the src.
- Removes the duplicate image as the placeholder and uses ImagePlaceholderAmp from Psammead instead.
- Image size is 70kB. Pageweight before change was 69.9kB. Pageweight after removing the image is 81.3kB, an increase of ~12kB.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/components/MediaPlayer/Amp/index.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { string, number } from 'prop-types';
 import { Helmet } from 'react-helmet';
-import { AmpImg } from '@bbc/psammead-image';
+import ImagePlaceholderAmp from '@bbc/psammead-image-placeholder';
```
Comment (jroebu14) [inline src/app/components/MediaPlayer/Amp/index.jsx]: Does this need the `Amp` suffix?

**Review by jroebu14 — APPROVED** [review]
Nice simple solution 🙂

File: src/app/components/MediaPlayer/Amp/index.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { string, number } from 'prop-types';
```
Comment (ryanmccombe) [inline src/app/components/MediaPlayer/Amp/index.jsx]:
```suggestion
import { string } from 'prop-types';
```

**Review by ryanmccombe — APPROVED** [review]
👍

File: src/app/components/MediaPlayer/Amp/index.jsx (hunk 2 above)
Comment (MuhammadAHussain) [reply src/app/components/MediaPlayer/Amp/index.jsx]: Done 👍

File: src/app/components/MediaPlayer/Amp/index.jsx (hunk 1 above)
Comment (MuhammadAHussain) [reply src/app/components/MediaPlayer/Amp/index.jsx]: Nope, not needed; removed it 👍

**Review by andrewscfc — COMMENTED** [review]
> Image size is 70kB. Bundle size before change was 69.9kB. Bundle size after removing image is 81.3kB, an increase of ~12kB.

I assume you are referring to pageweight here rather than bundlesize here; there are no javascript bundles on amp?

**Comment by MuhammadAHussain** [comment]
> > Image size is 70kB. Bundle size before change was 69.9kB. Bundle size after removing image is 81.3kB, an increase of ~12kB.
> 
> I assume you are referring to pageweight here rather than bundlesize here; there are no javascript bundles on amp?

Yup, my mistake; I amend the comment.

**Comment by andrewscfc** [comment]
> > > Image size is 70kB. Bundle size before change was 69.9kB. Bundle size after removing image is 81.3kB, an increase of ~12kB.
> > 
> > 
> > I assume you are referring to pageweight here rather than bundlesize here; there are no javascript bundles on amp?
> 
> Yup, my mistake; I amend the comment.

Thanks, just making sure I understood, or hadn't missed anything

**Review by ryanmccombe — APPROVED** [review]

**Pull request merged**